coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java

```diff
@@ -36,7 +36,7 @@
     
     public static final String BROWSER_PROTOCOL = "http";
     public static final String BROWSER_ADDRESS = "127.0.0.1";
-    public static final String PORTAL_ADDRESS = "kc-dev";
+    public static final String PORTAL_ADDRESS = "";
```

Comment:
I agree.  This doesn't seem right.  Why exactly don't the relative paths work?  I think this will break our other int tests